src/cix/css/Parser.hx

```diff
@@ -59,13 +59,18 @@ class Parser extends SelectorParser {
 
     var s = slice.toString();
     switch s.length {
-      case 3:
+      case 3 | 4:
```

Comment:
I wonder: is this working? I sometimes saw this interpreted as `3 | 4` (meaning `0`), but could've been older Haxe versions.